sdks/python/setup.py

```diff
@@ -229,6 +229,8 @@ def get_version():
     'selenium>=3.141.0,<4',
     'needle>=0.5.0,<1',
     'chromedriver-binary>=80,<81',
+    # use a fixed version of PIL for different python versions
+    'pillow==7.1.1',
```

Comment:
why?

Reply:
Python comes with PIL, but different versions of Python has different PIL versions.
To rule out the difference when taking screenshot, we use a fixed version of PIL explicitly for testing.

Reply:
Do we need to pin it? Pinning usually results in other issues. Maybe less so here because it is a test dependency.

Reply:
I've made it any version under a major version >=7,<8.